app/assets/stylesheets/components/search-results.scss

```diff
@@ -13,7 +13,8 @@
   margin-bottom: 0;
 }
 
-.documents-list h3 {
```

Comment:
Making this selector more specific so that Blacklight 8's CSS doesn't override it